tests/components/derivative/test_sensor.py

```diff
@@ -2,6 +2,7 @@
 from datetime import timedelta
 from unittest.mock import patch
 
+from homeassistant.const import TIME_HOURS, TIME_MINUTES, TIME_SECONDS
```

Comment:
`TIME_SECONDS` also on lines 83, 93, 113, 120, and 211? (Again, can't add a comment on those lines.)

Reply:
Changed them as well.